scout/commands/export/gene.py

```diff
@@ -22,15 +20,22 @@ def genes(build, json):
     """Export all genes from a build"""
     LOG.info("Running scout export genes")
     adapter = store
-
     result = adapter.all_genes(build=build)
+    gene_list = list(result)
+    if build == 'GRCh38':
```

Comment:
Why only for 'GRCh38' and not for '38'? That makes it a fairly hidden function. Do we need the old, broken '38' format for anything?

Reply:
I don't know. The default was that '37' and '38' behave in the same way and I didn't want to change it because I'm not sure if '38' is used for anything or in some other software..

Reply:
@henrikstranneheim: are you aware of anything downstream requiring the *broken* version of hg38 output? I'm not, and would prefer to replace it rather than risk confusion. Its quite ok to have both 38 and GRCh38 (and why not hg38) as options, but I think they should do the same thing.

Reply:
hg19 vs GRCh19 was another thing, but I hope we are beyond that at the 38 stage.. 😸

Reply:
Nothing I am aware of. To me it is fine to replace.
Any thoughts on upper case vs lower case in GRCh (or grch)?
Perfectly fine with hg38 as an alias. In MIP we are going to use grch38 internally (trying to avoid mixed case)

Reply:
grch is an acronym of the official name I think and hg is just short for human genome

Reply:
the acronym is going to be used in the CLI, while in the database the genome build field is named '38'. 
I could remove completely GRCh38 and leave only '38', as in was before. But it will export in the format 'chr1', 'chr1', 'chrX', 'chrM'. What do you think?

Reply:
@northwestwitch I say we like that, in principle! Just to be sure, what would happen if we re-import/upload the exported panel file (or import it into another instance of scout)?

@henrikstranneheim Yep, GRCh38 is Genome Reference Consortium human38. Generally, I try to preserve the case for established acronyms - some people go bonkers if you "misspell" their pet mixed-case software - but as long as the options are clearly stated in the docs everything is fine.

Reply:
> @northwestwitch I say we like that, in principle! Just to be sure, what would happen if we re-import/upload the exported panel file (or import it into another instance of scout)?
> 
In theory, if you export it using '38' you should be able to re upload it as it is. If you export a panel as GRCh38 then you have to reconvert it back to '38' ---> 'chrX' -> 'X' , 'chrM' -> 'M' otherwise you will get an error when you upload it (I guess)
I think we should leave the 38 thingy for now along with whatever we call the 'GRCh38' and then after we have fixed all the rest we can remove the 38 option.

Reply:
Sounds good!

Reply:
Ok, so what should I name this option, is 'GRCh38' fine?

Reply:
Yes, fine for now, then it can disappear when import behaviour has been fixed!